pkg/validator/server.go

```diff
@@ -43,16 +43,9 @@ func NewValidatingWebhook(webhookConfigName, osmNamespace, osmVersion, meshName
 		},
 	}
 
-	if enableReconciler {
-		// Create the ValidatingWebhook
-		if err := createValidatingWebhook(kubeClient, certificater, webhookConfigName, meshName, osmNamespace, osmVersion, validateTrafficTarget); err != nil {
-			return errors.Errorf("Error creating ValidatingWebhookConfiguration %s: %+v", webhookConfigName, err)
-		}
-	} else {
-		// Update the updateValidatingWebhookConfig with the OSM CA bundle, as the MutatingWebhook is created via Helm
-		if err := updateValidatingWebhookCABundle(webhookConfigName, certificater, kubeClient); err != nil {
-			return errors.Wrapf(err, "Error configuring ValidatingWebhookConfiguration %s", webhookConfigName)
-		}
+	// Create the ValidatingWebhook
+	if err := createValidatingWebhook(kubeClient, certificater, webhookConfigName, meshName, osmNamespace, osmVersion, validateTrafficTarget, enableReconciler); err != nil {
```

Comment:
same question as above. what happens if validating webhook already exists (osm control plane upgrade scenario)?

Reply:
same with this please check the newly added code in patch.go, it has code blocks if the webhook already exits